### PR TITLE
remove CacheFilterAttribute

### PR DESCRIPTION
**Pull request by SimonCropp**
it is not used

### CODE REVIEW

File: src/NuGetGallery/Filters/CacheFilter.cs [hunk 1]
```diff
@@ -1,42 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-using System;
-using System.Web;
-using System.Web.Mvc;
-
-namespace NuGetGallery.Filters
-{
-    public sealed class CacheFilterAttribute : ActionFilterAttribute
```
Comment (erdembayar) [inline src/NuGetGallery/Filters/CacheFilter.cs:9]: @NuGet/gallery-team 
Is it something we consume somewhere in our internal project via packaging or MEF?

Comment (SimonCropp) [reply src/NuGetGallery/Filters/CacheFilter.cs:9]: i doubt it, give NuGetGallery does not produce a nuget

**Review by erdembayar — APPROVED** [review]

**Comment by SimonCropp** [comment]
can this be merged?

**Comment by erdembayar** [comment]
@SimonCropp 
CI build fails. It fails missing dependency to `Gallery.GalleryTools.nuspec`

**Review by erdembayar — CHANGES_REQUESTED** [review]
Code doesn't compile anymore.

**Comment by SimonCropp** [comment]
fixed

**Pull request closed without merging**